### PR TITLE
fix: stepper UI fix on Home page

### PR DESCRIPTION
**Pull request by gitanjli525**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [x] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Checklist

- [x] I ran `npm run re:build`
- [x] I reviewed submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

File: src/screens/HyperSwitch/Home/HomeV2.res [hunk 1]
```diff
@@ -37,13 +41,17 @@ module HomePageHorizontalStepper = {
...
+              {index < step
+                ? <Icon name="check" size=12 customIconColor="#006DF9" />
+                : (index + 1)->Int.toString->React.string}
             </span>
-            <UIUtils.RenderIf condition={index !== stepperItemsArray->Array.length - 1}>
+            <UIUtils.RenderIf condition={index <= stepperItemsArray->Array.length - 1}>
               <div className="relative w-full">
                 <div className={`absolute h-1 rounded-full z-1 ${index->getProgressBarStyle}`} />
-                <div className="w-full h-1 rounded-full bg-grey-700 bg-opacity-10" />
+                {index != stepperItemsArray->Array.length - 1
```
Comment (PritishBudhiraja) [inline src/screens/HyperSwitch/Home/HomeV2.res:52]: Use UIUTils.renderif

File: src/screens/Home/HomeV2.res [hunk 2]
```diff
@@ -17,13 +17,17 @@ module HomePageHorizontalStepper = {
     }
 
     let getStepperStyle = index => {
-      index <= step ? "bg-blue-700 text-white border-transparent" : "border-blue-700 text-blue-700 "
+      if index < step {
+        "bg-white text-pink border-blue-700"
```
Comment (PritishBudhiraja) [inline src/screens/Home/HomeV2.res:21]: Are we sure with this color?

**Review by PritishBudhiraja — APPROVED** [review]

File: src/screens/Home/HomeV2.res [hunk 3]
```diff
@@ -37,13 +41,17 @@ module HomePageHorizontalStepper = {
         <div className="flex flex-col gap-2.5 w-full" key={index->Int.toString}>
           <div className="flex items-center gap-2">
             <span
-              className={`h-6 w-6 flex items-center justify-center border-2 rounded-md font-semibold ${index->getStepperStyle} ${getTextStyle}`}>
-              {(index + 1)->Int.toString->React.string}
+              className={`h-6 w-7 flex items-center justify-center border-1 rounded-md font-semibold ${index->getStepperStyle} ${getTextStyle}`}>
+              {index < step
```
Comment (Riddhiagrawal001) [inline src/screens/Home/HomeV2.res:45]: Can we use UIUtils.RenderIf instead of ternary operator here ?

**Review by PritishBudhiraja — APPROVED** [review]

File: src/screens/Home/HomeV2.res [hunk 4]
```diff
@@ -37,13 +41,20 @@ module HomePageHorizontalStepper = {
         <div className="flex flex-col gap-2.5 w-full" key={index->Int.toString}>
           <div className="flex items-center gap-2">
             <span
-              className={`h-6 w-6 flex items-center justify-center border-2 rounded-md font-semibold ${index->getStepperStyle} ${getTextStyle}`}>
-              {(index + 1)->Int.toString->React.string}
+              className={`h-6 w-7 flex items-center justify-center border-1 rounded-md font-semibold ${index->getStepperStyle} ${getTextStyle}`}>
+              <UIUtils.RenderIf condition={index < step}>
+                <Icon name="check" size=12 customIconColor="#006DF9" />
```
Comment (JeevaRamu0104) [inline src/screens/Home/HomeV2.res:46]: move this color code tailwind file

Comment (gitanjli525) [reply src/screens/Home/HomeV2.res:46]: we are unable to access the tailwind config file here, so we need to use direct color here

File: src/screens/Home/HomeV2.res (hunk 3 above)
Comment (gitanjli525) [reply src/screens/Home/HomeV2.res]: @Riddhiagrawal001 , updated the code with the suggestion

**Review by JeevaRamu0104 — APPROVED** [review]

**Pull request merged**